pom.xml

```diff
@@ -182,6 +179,12 @@
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-stream-binder-rabbit</artifactId>
             <scope>runtime</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Why do you need to exclude that ?

Reply:
Because commons-logging is another logging library of apache, like log4j and logback, and we use logback for our logs. I exclude it for not having two logging libraries in the console, but it will need another issue to verify and uniform slf4j bridges.

It come from apache httpclient used by Spring-cloud-stream.  
![image](https://github.com/user-attachments/assets/afb7359a-74be-44df-ac92-76e7a0a67fa4)

Reply:
Yes but rabbit-binder is imported in a lot of our repositories, why is it a problem only for this one ?